client-app/build.sbt

```diff
@@ -0,0 +1,26 @@
+ThisBuild / scalaVersion := "2.12.12"
```

Comment:
Since you've just finished upgrading to Daml 1.10, you'll be happy to know that we (by which I mean @cocreature) have recently completed the transition of our Scala codebase to 2.13 and are planning to drop 2.12 soon-ish.

Reply:
This hasn’t made it into 1.10 so you’ll have to wait until 1.11 (or use a snapshot) before migrating.

Reply:
I'll stick with 2.12.12 for time being and 1.10